src/index.html

```diff
@@ -7,8 +7,67 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="/images/logo.png"
+          alt="moyoLogo"
+          class="icon"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <a href="#" class="nav__link is-active">
+            <li class="nav__item">
```

Comment:
place the `<a>` tag inside the` <li>` tag, not the other way around. So, the structure should be: 
`<li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>`.